centreon/www/front_src/src/Dashboards/Dashboard/Layout/Panel/MorePanelActions.tsx

```diff
@@ -67,7 +69,7 @@ const MorePanelActions = ({
     close();
   };
 
-  const displayEditButtons = canEdit && isEditing;
+  const displayEditButtons = canEdit;
```

Comment:
```suggestion
  const displayEditButtons = canEdit && isEditing;
```

Reply:
This condition must remain

Reply:
You have the `viewActions`. You add a condition on the label and the icon to change it following role